File: fixtures/r_output/post_json_multiple_headers.R
require(httr)

headers = c(
  `Content-Type` = 'application/json',
  `X-API-Version` = '200'
)

data = '{"userName":"username123","password":"password123", "authLoginDomain":"local"}'

res <- httr::POST(url = 'https://0.0.0.0/rest/login-sessions', httr::add_headers(.headers=headers), body = data, config = httr::config(ssl_verifypeer = FALSE))

File: test/fixtures/ruby-httparty/post_json_multiple_headers.rb
require 'httparty'
require 'json'

url = 'http://localhost:28139/rest/login-sessions'
headers = {
  'Content-Type': 'application/json',
  'X-API-Version': '200',
}
# The object won't be serialized exactly like this
# body = '{"userName":"username123","password":"password123", "authLoginDomain":"local"}'
body = {
  'userName' => 'username123',
  'password' => 'password123',
  'authLoginDomain' => 'local'
}.to_json
res = HTTParty.post(url, headers: headers, body: body, verify: false)
